cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md

```diff
@@ -170,14 +170,14 @@ However, if you are interested in finding hints, tips and guidance for conductin
 
 <center>–––</center>
 
-> "I want to use the public analysis example from the MIT jet-analysis group's papers."
+> "I want to learn about the published jet-analysis papers by the MIT group."
 
 - See *Jet Substructure Studies with CMS Open Data*, A. Tripathee et al., Apr 19, 2017, MIT-CTP-4890, [arXiv:1704.05842](https://inspirehep.net/record/1593756),
 - and *Exposing the QCD Splitting Function with CMS Open Data*, A. Larkoski et al., Apr 17, 2017, MIT-CTP-4891, [arXiv:1704.05066](https://inspirehep.net/record/1591972).
 
 <center>–––</center>
 
-> "I want to (re)validate the 2010 datasets within my setup…"
+> "I want to run the examples used for 2010 datasets validation within my setup…"
```

Comment:
"I want to run the examples used for validation of the 2010 datasets within my setup…"